opal/mca/common/ucx/common_ucx_wpool.h

```diff
@@ -470,7 +470,6 @@ opal_common_ucx_wpmem_cmpswp_nb(opal_common_ucx_wpmem_t *mem, uint64_t compare,
     rc = _periodical_flush_nb(mem, winfo, target);
     if(OPAL_UNLIKELY(OPAL_SUCCESS != rc)){
         MCA_COMMON_UCX_VERBOSE(1, "_incr_and_check_inflight_ops failed: %d", rc);
-        return rc;
```

Comment:
should not we also `goto` line 474 if `opal_common_ucx_atomic_cswap_nb` completed immediately (which is possible in case of error)?

Reply:
in case if more actions are added after this check - yes, we have to jump, but for now it is not needed

Reply:
it does `_periodical_flush_nb`
i meant checking result of `opal_common_ucx_atomic_cswap_nb`, not the line you fixed

Reply:
not sure if such jump doesn't break anything else